calendar.music

```diff
@@ -470,7 +470,7 @@
 11/23	Thomas Tallis dies, 1585
 11/24	Scott Joplin born near Linden, Texas, 1867
 11/25	"The Last Waltz" concert is played by The Band at Winterland, 1976
-11/25	Johann Strauss, Jr., writes 'On the Beautiful Blue Danube', 1867
+11/25	Johann Strauss II, writes 'On the Beautiful Blue Danube', 1867
```

Comment:
After a quick research I have found around both "Johann Strauss Jr." and "Johann Strauss II" (and also "the Younger", "the Son"). Why do you recommend using "Johann Strauss II"?

If we were to keep the Jr version, I would remove the comma between "Strauss" and "Jr.".

Reply:
Chicago manual of style has recommended a comma before Jr, but not before II or III, etc. I learned that in school. However, since its 93 edition, the CMA has stopped recommending that. The New Yorker style guide, however, continues to recommend a comma.

Reply:
Thanks, I did not know about this.

According to `grep -R "Jr"`, at the moment we have 4 occurences of Jr with a comma and 1 without it, so I would say that keeping the comma is more consistent. However, since we have only 5 occurences of Jr, we can also easily change the repository style, if we wanted. In any case it would be nice to adopt the same style in all cases.

Reply:
> After a quick research I have found around both "Johann Strauss Jr." and "Johann Strauss II" (and also "the Younger", "the Son"). Why do you recommend using "Johann Strauss II"?

Familiarity and because I never came across Jr. But you sent me digging, and searching for "Johann Strauss II,Johann Strauss Sohn,Johann Strauss Jr,Johann Strauss Junior" in Google Ngram's German corpus only shows Sohn used at all, so I would use "Son" on second thought. With or without a comma I can't say. And I don't think we use any style guide in the sense of adopting substantial chunks of it. Since these run to hundreds of pages (over 1000 for the Chicago Manual of Style IIRC), I'm fine with that.+